packages/fast-usdc/src/exos/transaction-feed.js

```diff
@@ -148,18 +155,20 @@ export const prepareTransactionFeedKit = (zone, zcf) => {
 
         /** @param {string} operatorId */
         removeOperator(operatorId) {
-          const { operators } = this.state;
+          const { operators, pending, risks } = this.state;
           trace('removeOperator', operatorId);
           const operatorKit = operators.get(operatorId);
           operatorKit.admin.disable();
           operators.delete(operatorId);
+          pending.delete(operatorId);
+          risks.delete(operatorId);
```

Comment:
I suppose this change is covered by the revised `remove operator` test, though I haven't studied it well enough to be certain.